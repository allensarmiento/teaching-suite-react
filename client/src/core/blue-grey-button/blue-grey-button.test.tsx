import { shallow } from 'enzyme';
import BlueGreyButton from './blue-grey-button.component';

it('renders component', () => {
  expect(shallow(<BlueGreyButton>Button</BlueGreyButton>)).toMatchSnapshot();
});

it('renders component with className props', () => {
  expect(shallow(<BlueGreyButton className="large">Button</BlueGreyButton>))
    .toMatchSnapshot();
});

it('renders component with type props', () => {
  expect(shallow(<BlueGreyButton type="button">Button</BlueGreyButton>))
    .toMatchSnapshot();
});
