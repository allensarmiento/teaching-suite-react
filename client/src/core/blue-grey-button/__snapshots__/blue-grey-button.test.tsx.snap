// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders component 1`] = `
<button
  className="button"
>
  Button
</button>
`;

exports[`renders component with className props 1`] = `
<button
  className="large button"
>
  Button
</button>
`;

exports[`renders component with type props 1`] = `
<button
  className="button"
  type="button"
>
  Button
</button>
`;
